packages/form-js/test/spec/Form.spec.js

```diff
@@ -11,6 +11,11 @@ import {
   isSingleStart
 } from '../TestHelper';
 
+import {

```

Comment:
What's going on here?

Reply:
What do you think is going on?

Reply:
I don't understand why we have to import chai here. 🤔